assembl/static/js/app/views/tokenVoteSession.js

```diff
@@ -252,8 +252,15 @@ var RemainingCategoryTokensView = Marionette.ItemView.extend({
     el.addClass("description");
 
     var categoryNameLangString = category.get("name");
-    var categoryNameBestTranslation = categoryNameLangString.bestWithErrors(that.userLanguagePreferences, false).entry.value();
-    // var s = i18n.sprintf(i18n.ngettext("<span class='available-tokens-number'>%d</span> remaining %s token", "<span class='available-tokens-number'>%d</span> remaining %s tokens", data["remaining_tokens"]), data["remaining_tokens"], category.get("typename")); // TODO: use "name" field instead (LangString)
+    var categoryNameBestTranslation = "";
+    try {
+      categoryNameBestTranslation = categoryNameLangString.bestWithErrors(that.userLanguagePreferences, false).entry.value();
+    }
+    catch (e) {
+      console.error("error managing langstring categoryNameLangString: ", e, "langstring was:", categoryNameLangString);
+      // TODO: understand and fix the root cause of the problem (happens only when displayed in a modal)
```

Comment:
What is the error here? Why would you have to resort to this?

Reply:
I could not find the origin of the issue and it was very difficult to reproduce (rare).